week-1/L-functions-nested/exercise.js

```diff
@@ -3,3 +3,28 @@ var mentor2 = "Irina";
 var mentor3 = "Mimi";
 var mentor4 = "Rob";
 var mentor5 = "Yohannes";
+//The percentage function
+function percentage(numberOfStudents, numberOfMentors) {
+  var total = numberOfStudents + numberOfMentors;
+  var percentageOfStudents = (numberOfStudents / total) * 100;
+  var percentageOfMentors = (numberOfMentors / total) * 100;
+  printPercentage(percentageOfStudents, percentageOfMentors);
+}
+
+//print the percentage message
+function printPercentage(percentageOfStudents, percentageOfMentors) {
+  console.log("Percentage students: " + Math.round(percentageOfStudents) + "%");
+  console.log("Percentage mentors: " + Math.round(percentageOfMentors) + "%");
+}
+
+//prints the metors greeting
+function printGreeting(name) {
+  console.log("Hello " + name.toUpperCase());
+}
+
+percentage(15, 8);
```

Comment:
After you have time to think about my previous comment, maybe you could address this function as well - `percentage` as a name doesn't really give away what it really does :)

Reply:
Thank you for your kind comments. You have given me a lot to work with.